app/src/app_common.cc

```diff
@@ -289,14 +289,14 @@ App* AddApp(App* app, std::map<std::string, InitResult>* results) {
     assert(!g_default_app);
     g_default_app = app;
   }
-  UniquePtr<AppData> app_data = MakeUnique<AppData>();
+  std::unique_ptr<AppData> app_data = std::make_unique<AppData>();
   app_data->app = app;
   app_data->cleanup_notifier.RegisterOwner(app);
   if (!g_apps) {
-    g_apps = new std::map<std::string, UniquePtr<AppData>>();
+    g_apps = new std::map<std::string, std::unique_ptr<AppData>>();
     created_first_app = true;
   }
-  (*g_apps)[std::string(app->name())] = app_data;
```

Comment:
Because the old UniquePtr actually had a copy constructor (which technically acted as a move constructor 😱), there are quite a few places where the code had to be modified to perform an explicit move where it wasn't happening before.